models/commons/package.json

```
@@ -8,18 +8,28 @@
     "url": "https://github.com/commercetools/test-data.git",
     "directory": "models/commons"
   },
-  "keywords": ["javascript", "typescript", "test-data"],
+  "keywords": [
+    "javascript",
+    "typescript",
+    "test-data"
+  ],
   "license": "MIT",
   "publishConfig": {
     "access": "public"
   },
   "main": "dist/commercetools-test-data-commons.cjs.js",
   "module": "dist/commercetools-test-data-commons.esm.js",
-  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
   "dependencies": {
     "@babel/runtime": "^7.17.9",
     "@babel/runtime-corejs3": "^7.17.9",
     "@commercetools-test-data/core": "10.8.1",
+    "@commercetools-test-data/product-discount": "workspace:^",
```

We should be following the same pattern as the other repo dependencies:
```suggestion
    "@commercetools-test-data/product-discount": "10.8.1",
```